BUILD

```diff
@@ -15,6 +15,30 @@ exports_files([
 # @external_begin
 filegroup(
     name = "node_modules",
-    srcs = glob(["node_modules/**/*"]),
+    srcs = glob(
+        # Only include files we might use, to reduce the file count and surface size of 
+        # filename problems.
+        [
+            "node_modules/**/*.js",
+            "node_modules/**/*.json",
+            "node_modules/**/*.d.ts",
+        ],
+        exclude = [
+            # e.g. node_modules/adm-zip/test/assets/attributes_test/New folder/hidden.txt
+            "node_modules/**/test/**",
+            # e.g. node_modules/xpath/docs/function resolvers.md
+            "node_modules/**/docs/**",
+            # e.g. node_modules/puppeteer/.local-chromium/mac-536395/chrome-mac/Chromium.app/Contents/Versions/66.0.3347.0/Chromium Framework.framework/Chromium Framework
+            "node_modules/**/.*/**",
+            # Ignore paths with spaces.
+            "node_modules/**/* *",
+        ],
+    ) + glob(["node_modules/.bin/*"]),
 )
 # @external_end
+
+load("@com_github_bazelbuild_buildtools//buildifier:def.bzl", "buildifier")
+
+buildifier(
```

Comment:
does this work all the way now?
I'm curious if it picks up .bzl files too
If this approach works well, I might want to update my other repos

Reply:
It worked locally when I hacked buildifier to work on windows at all. 

Can I ask you to test though? I haven't tested on linux/osx. Plus the hacks to make it work on windows involve setting absolute paths and it's always a bit of a hassle.

Reply:
I tried testing it and it made edits.
Can we move buildifier to a separate PR?